src/action_management/include/actionmanagement.h

```diff
@@ -0,0 +1,339 @@
+#ifndef _ACTIONS_H_
+#define _ACTIONS_H_ 
+
+#include <stdio.h>
+
+
+/* An enumeration of all supported actions. */
+/* All synonyms will be provided to CLI. */
+enum actions {
+  OPEN, /* world action */
+  CLOSE, /* world action */
+  PUSH, /* world action */
+  PULL, /* world action */
+  EXAMINE, /* synonymous to "look at", is a world action */
+  TURN_ON, /* world action */
+  TURN_OFF, /* world action */
+  TALK_TO, /* NPC action */
+  GIVE, /* requires 2 arguments,  NPC action */
+  GO, /* world action */
+  TAKE, /* synonymous with "pick up", is an inventory action */
+  DROP, /* inventory action */
+  CONSUME, /* synonymous with "use", "drink", "eat", is an inventory action */
+  USE_ON, /* world and NPC action */
```

Comment:
Tried to change the description of USE_ON to match that of GIVE
```suggestion
  USE_ON, /* requires two arguments separated by 'on', world and NPC action  */
```